refarch-integrations/refarch-cosys-integration/refarch-cosys-integration-starter/src/main/resources/application-cosys-client.yml

```diff
@@ -0,0 +1,17 @@
+spring:
+  security:
+    oauth2:
+      client:
+        provider:
+          cosys:
+            issuer-uri: ${SSO_COSYS_ISSUER_URL}
+            user-info-uri: ${SSO_COSYS_ISSUER_URL}/protocol/openid-connect/userinfo
+            jwk-set-uri: ${SSO_COSYS_ISSUER_URL}/protocol/openid-connect/certs
+            user-name-attribute: user_name
+        registration:
+          cosys:
+            provider: cosys
+            authorization-grant-type: client_credentials
+            client-id: ${SSO_COSYS_CLIENT_ID}
+            client-secret: ${SSO_COSYS_CLIENT_SECRET}
+            scope: email, profile, openid  # needed for userInfo endpoint
```

Comment:
would it be better to move that to the documentation, like you did it for the s3?

Reply:
Ah yeah good idea. Was about to look for this 👍

Reply:
Done